.github/workflows/load-generate.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: load-generate
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.11'
+
+      - run: |
+          pip install -r requirements.txt
+
+      - run: python compile.py
+
+      - run: |
```

Comment:
@uchitsa what this is for?

Reply:
for exec python script

Reply:
@uchitsa I still don't understand what is the purpose of this line. You fetch all tags, then you assign the name of the latest tag to the `latest` environment variable, and then you forget it (since the command finishes). Why?